pkg/network/common/egressip.go

```diff
@@ -90,24 +97,27 @@ func NewEgressIPTracker(watcher EgressIPWatcher) *EgressIPTracker {
 	return &EgressIPTracker{
 		watcher: watcher,
 
-		nodes:            make(map[ktypes.UID]*nodeEgress),
-		nodesByNodeIP:    make(map[string]*nodeEgress),
+		nodes:            make(map[ktypes.UID]*NodeEgress),
+		nodesByNodeIP:    make(map[string]*NodeEgress),
```

Comment:
(It would be nice to have a separate commit for renaming/exporting NodeEgress, so that all the parts that are just capitalization are separate from the stuff that actually changes functionality)

Reply:
Done